plugins/PTCTools/src/main/java/org/micromanager/ptctools/PtcToolsExecutor.java

```diff
@@ -249,19 +248,20 @@ public void doSequence(JLabel resultLabel) {
    
    
    private void showDialog(final String label, final PtcSequenceRunner sr) {
-      final MMFrame dialog = new MMFrame();
+      final JFrame dialog = new JFrame();
       dialog.setBounds(settings_.getInteger(PtcToolsTerms.WINDOWX, 100), 
               settings_.getInteger(PtcToolsTerms.WINDOWY, 100), 400, 100);
       dialog.addComponentListener(new ComponentAdapter() {
          @Override
          public void componentMoved(ComponentEvent e) {
-            dialog.savePosition();
+            WindowPositioning.setUpBoundsMemory(dialog, dialog.getClass(), null);
          }
       });
```

Comment:
This should not be necessary, because `setUpBoundsMemory()` called below should make the necessary arrangements.